### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vadPavlenko**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+
+    def wrapper(*args) -> Any:
+
+        if args in store_result:
+            print("Getting from cache")
+            return store_result[args]
+        else:
+            print("Calculating new result")
+
+        store_result[args] = func(*args)
```
Comment (vasylhnatiuk) [inline app/main.py]:
```suggestion
            store_result[args] = func(*args)
```

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    store_result = {}
+
+    def wrapper(*args) -> Any:
+
+        if args in store_result:
+            print("Getting from cache")
+            return store_result[args]
```
Comment (vasylhnatiuk) [inline app/main.py]:
```suggestion
```

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
use only one return in wrapper()

**Comment by vadPavlenko** [comment]
Created a new pull-request. #1002. Can you check it again?

**Pull request closed without merging**